PluginCore/PluginCore/Helpers/PathHelper.cs

```diff
@@ -19,7 +20,7 @@ public static String BaseDir
         {
             get
             {
-                if (PluginBase.MainForm.StandaloneMode) return AppDir;
+                if (PluginBase.MainForm.StandaloneMode) return Path.GetDirectoryName(GetAssemblyPath(Assembly.GetExecutingAssembly()));
```

Comment:
In this case it returns the original path of PluginCore.dll (avoiding the path used when shadow copying), so yeah, it can return a different path than the application directory. In FD they are in the same folder, but if another application uses PluginCore.dll, it won't be the folder where the application resides, but then again, that's why there is an AppDir, this is BaseDir.

I don't see the point of compiling this code only for testing.